addons/mail/controllers/discuss.py

```diff
@@ -414,6 +414,8 @@ def channel_ping(self, channel_id, rtc_session_id=None, check_rtc_session_ids=No
     def mail_thread_data(self, thread_model, thread_id, request_list, **kwargs):
         res = {}
         thread = request.env[thread_model].with_context(active_test=False).search([('id', '=', thread_id)])
+        if not thread:
+            return res
```

Comment:
Hello, this seems strange, why is the RPC done on a non-existing thread in the first place?

Reply:
@seb-odoo , It's strange but here is the specific case where `thread_id` exist but while [searching](https://github.com/odoo/odoo/pull/91728/files#diff-7bd6aab640f703675328c028f79f8c1775d07bbecc7e7e250894227cabb4e299R416) in the thread model it will not be able to find the thread due to the access right. that's why we have to return it.

For Ex: create a private task assigned to users A(current user) & B, remove the current user from the assignees > remove the current user from the followers > traceback because the access right of thread and task is removed but still the thread_id exist.

Thanks

Reply:
But isn't this a more general problem with the form view? If the current user can no longer access that thread/task, fixing the chatter RPC this is just fixing a symptom, the correct fix should be that the record is no longer displayed on the form view instead.

Reply:
@seb-odoo , As we are already on that form view so once the follower is removed it will display the access error dialog(already handled) but on the removal of the follower it will reload the chatter and at that time it was giving the symptoms(traceback).  

As we are on the same form view and the record is already loaded so I don't think it's a proper solution to move to the next record or open the previous view. 

Thanks

Reply:
@seb-odoo , done the changes. thanks